### PR TITLE
Add artists view ld8

### PR DESCRIPTION
**Pull request by ashleyjelks**
screen shot 2018-10-30 at 2 57 30 pm
Implement Artwork grid in Show View with Infinite Scroll.

[Links to LocalDiscovery 8 in Jira](https://artsyproduct.atlassian.net/browse/LD-8)

### CODE REVIEW

File: Pod/Classes/ViewControllers/ARArtworkComponentViewController.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#import "ARComponentViewController.h"
```
Comment (orta) [inline Pod/Classes/ViewControllers/ARArtworkComponentViewController.h]: This is only needed for *root* components, you should only need these type of changes if you're planning to replace `/artworks/:id` for all of artsy

File: src/lib/Scenes/Show/Components/Artworks/index.tsx [hunk 2]
```diff
@@ -1,14 +1,25 @@
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
 import React from "react"
-import { Text } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
 
 interface Props {
-  artworks: any[]
+  show: any
```
Comment (l2succes) [inline src/lib/Scenes/Show/Components/Artworks/index.tsx]: We should actually use the `Artworks_show` type here

File: Pod/Classes/ViewControllers/ARArtworkComponentViewController.h (hunk 1 above)
Comment (ashleyjelks) [reply Pod/Classes/ViewControllers/ARArtworkComponentViewController.h]: This is resolved!

**Comment by l2succes** [comment]
👍  Muy bien

**Pull request merged**